Algorithmia/Algorithm/AlgoOption.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AlgoOptions.swift
+//  AlgorithmiaSwiftDemo
+//
+//  Created by Erik Ilyin on 11/8/16.
+//  Copyright © 2016 algorithmia. All rights reserved.
+//
+
+import Foundation
+
+enum AlgoOption {
+    case Timeout(Int)
+    case Stdout(Bool)
+    case RawOutput
+    case VoidOutput
```

Comment:
`output=void` affects the response format (there is no way to get the algorithm result), and the successful response will be something like: 
```
{"async":"void","request_id":"req-d898c234-08fc-4755-84ea-f745f2a2a146"}
```

so you would probably need a separate `AsyncResponse` type that lets you get the request ID. You might take a look at how this was done in Go: https://github.com/algebraic-brain/algorithmia-go/blob/master/algorithm.go#L91